src/components/OptionsSelector.js

```diff
@@ -191,11 +191,16 @@ class OptionsSelector extends Component {
             CTRLEnterConfig.shortcutKey,
             () => {
                 const focusedOption = this.state.allOptions[this.state.focusedIndex];
```

Comment:
NAB if this variable is only needed if `this.props.canSelectMultipleOptions` is false then we should move it after that check.

Reply:
Updated